media/docs/cute/01_layout.md

```diff
@@ -165,7 +165,7 @@ auto layout_2sx4s = make_layout(make_shape(Int<2>{},Int<4>{}));
 auto layout_2sx4d = make_layout(make_shape(Int<2>{},4));
 
 auto layout_2x4 = make_layout(make_shape (2, make_shape (2,2)),
-                              make_stride(4, make_stride(1,2)));
+                              make_stride(4, make_stride(2,1)));
```

Comment:
I think, it was a typo. Because when I initialize the `layout_2x4` with `make_stride(1,2)` (as suggested in the original), the results I'm getting from the `print2D` and `print1D` are:

```
> print2D(layout_2x4)
  0   1   2   3
  4   5   6   7
```

```
> print1D(layout_2x4)
  0   1   2   3   4   5   6   7
```

These results are different from the ones shown further in the [Using a Layout](https://github.com/NVIDIA/cutlass/blob/main/media/docs/cute/01_layout.md#using-a-layout) section for the `layout_2x4`:

```
> print2D(layout_2x4)
  0   2   1   3
  4   6   5   7
```

```
> print1D(layout_2x4)
  0   4   2   6   1   5   3   7
```

I believe, the reason is that implicit indexing into the nested layout in the second mode is in the "generalized-column-major" order. So, if the nested stride of the second mode is in column-major, we see the resulting values in the consecutive order instead of the shown permuted one. Switching from the column-major to the row-major stride of the nested layout makes the `print2D` and `print1D` outputs consistent with the ones shown in the [Using a Layout](https://github.com/NVIDIA/cutlass/blob/main/media/docs/cute/01_layout.md#using-a-layout) section.

Reply:
Thanks for explaining!  I missed that print statement later on in the tutorial.